### DISCUSSION TITLE
Doubleheader Weeks - Bypassing weekValidation.py?

### DISCUSSION THREAD

**Discussion by Dstamper89** [joeyagreco/leeger#102] [Q&A]
The two leagues I participate in on MyFantasyLeague utilize doubleheader weeks throughout the season. I do not believe "weekValidation.py" allows for this and returns an error. My understanding of how the program works, it won't load any league data unless it passes all of the validation checks. Is this check crucial to the functionality of the program, or can it be easily bypassed to allow for these doubleheader weeks to run without error?

**Comment by joeyagreco** [comment]
Thanks for checking out the library!

There's a lot of niche features that this library does not (yet) support simply for lack of real leagues to test them on. 

I currently do not have info for an MFL league that uses doubleheader weeks to test on.

If you'd like to send me your league info or hop on a call sometime I'd be happy to look into this so I can add support for it in this library.

Either way, I just added support for disabling validation when loading a league [in this release](https://github.com/joeyagreco/leeger/releases/tag/v2.3.0).

Basically, just pass `validate=False` into the `loadLeague()` method.

I'm not sure that disabling this will help you get much further though, as the league data has to be loaded in a very particular way in order to generate all of the stats accurately.

Let me know if that helps. Either way, I'm happy to work on adding this feature to the library, though I would need your league info to do so.

**Reply by Dstamper89** [reply]
I understand. In order to take a look at my league, do you need my login info? I'm not sure I'm too comfortable including that, but if it's possible without that, I can provide you with the League ID. I appreciate the quick response and I love the potential of this program. I understand this is a fringe case, but it's really neat to see such a thorough response so quickly.

I'm a relative novice when it comes to coding so I'm not too much help there, but if I can get you information to help, please let me know.

**Reply by joeyagreco** [reply]
So to access the league I would need the info [here ](https://github.com/joeyagreco/leeger/blob/main/doc/league_loader/my_fantasy_league.md) (would include username/password)

It's totally fine if you're not comfortable with that.

The 2 ways I _have_ gone about resolving issues like this for this library in the past are:

1. Get on a call with the person having the issue and see if I can resolve the issue (or see _how_ to resolve the issue) on the call.
2. Get the info needed to run the code locally on my machine to resolve the issue.

Either way works for me, if you're not comfortable with either of those I understand.

If you would like to try either of those, just shoot me an email: joeyagreco@gmail.com
